src/functionalTest/resources/features/F-008 - Create PRL Staff and Judicial Role Assignments/F-008.feature

```diff
@@ -62,3 +62,85 @@ Feature: F-008 : Create Role Assignments for PrivateLaw Caseworker and Judicial
     Then a positive response is received,
     And the response has all other details as expected
     And a successful call [to delete existing role assignments corresponding to the test actorId] as in [S-074_DeleteDataForRoleAssignments].
+
+  @S-008.04
+  @FeatureToggle(DB:privatelaw_wa_1_5=on) @FeatureToggle(EV:CASEWORKER_FTA_ENABLED=on)
+  Scenario: must successfully create org role mapping for Hearing Centre Administrator
+    Given a user with [an active IDAM profile with full permissions],
+    And a successful call [to verify caseworker details for Hearing Centre Administrator (ABA5 PRL)] as in [S-008.04__VerifyCaseworkerDetails],
+    And a successful call [to delete existing role assignments corresponding to the test actorId] as in [DeleteDataForRoleAssignments],
+    And a successful call [to publish existing CRD user ids to endpoint] as in [F-008_PushMessageToCRDService],
+    And the request [contains the actorId of the user just published who has Hearing Centre Administrator role],
+    When a request is prepared with appropriate values,
+    And it is submitted to call the [Fetch Assignment From Role Assignment Service] operation of [Role Assignment Service],
+    Then a positive response is received,
+    And the response has all other details as expected
+    And a successful call [to delete existing role assignments corresponding to the test actorId] as in [DeleteDataForRoleAssignments].
+
+@S-008.04b
+@FeatureToggle(DB:privatelaw_wa_1_5=on) @FeatureToggle(EV:CASEWORKER_FTA_ENABLED=on)
+Scenario: must successfully create org role mapping for Hearing Centre Administrator + Case allocator
+    Given a user with [an active IDAM profile with full permissions],
+    And a successful call [to verify caseworker details for Hearing Centre Administrator + Case allocator (ABA5 PRL)] as in [S-008.04b__VerifyCaseworkerDetails],
+    And a successful call [to delete existing role assignments corresponding to the test actorId] as in [DeleteDataForRoleAssignments],
+    And a successful call [to publish existing CRD user ids to endpoint] as in [F-008_PushMessageToCRDService],
+    And the request [contains the actorId of the user just published who has Hearing Centre Administrator role + Case allocator],
+    When a request is prepared with appropriate values,
+    And it is submitted to call the [Fetch Assignment From Role Assignment Service] operation of [Role Assignment Service],
+    Then a positive response is received,
+    And the response has all other details as expected
+    And a successful call [to delete existing role assignments corresponding to the test actorId] as in [DeleteDataForRoleAssignments].
```

Comment:
Can you put the new caseworker tests in numerical order: i.e. move 04 & 04b after 02 and 02b.

Reply:
Resolved in 546c3beb1a072096af71bde8caf4c52c03b46787.